application/src/main/java/org/opentripplanner/routing/algorithm/mapping/RouteRequestToFilterChainMapper.java

```diff
@@ -3,6 +3,7 @@
 import java.time.Duration;
 import java.time.Instant;
 import java.util.List;
+import java.util.OptionalInt;
```

Comment:
Probably unused import.

Reply:
I removed the unused imports in this file